src/utils/service_builder.rs

```diff
@@ -53,6 +53,9 @@ const WIRE_PROTOCOL_VERSION_MAJOR: u8 = 1;
 /// Default value for the limit on the request body size (in bytes) - equal to 1MB
 const DEFAULT_BODY_LEN_LIMIT: usize = 1 << 19;
```

Comment:
We should probably fix this one as well 👀

Reply:
Yeah, was going to do that in a separate patch. Let me do that now...

Reply:
[Speedy PR](https://github.com/parallaxsecond/parsec/compare/master...joechrisellis:body_len_limit_fix?expand=1)!